Sources/Hyperspace/Service/Backend/BackendServiceHelper.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  BackendServiceHelper.swift
+//  Hyperspace
+//
+//  Created by Earl Gaspard on 5/9/18.
+//  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+
+/// A helper struct for use with BackendService.
+public struct BackendServiceHelper {
+    
+    /// Attempts to transform response data into the associated response model.
+    ///
+    /// - Parameters:
+    ///   - data: The raw Data retrieved from the network.
+    ///   - request: The NetworkRequest to be executed.
```

Comment:
I might tweak the comment here slightly since this isn't actually executing the `NetworkRequest`:
`- request: The NetworkRequest to be executed.`
to
`- request: The NetworkRequest that will be used to transform the Data.`